src/test/java/org/alfresco/repo/admin/Log4JHierarchyInitTest.java

```diff
@@ -63,4 +73,38 @@ public void testAddingLog4jProperties() throws Throwable
         // We expect DEBUG to be on
         assertTrue("DEBUG was not enabled for logger " + this.getClass(), log.isDebugEnabled());
     }
+
+    public void setUpStreams() throws UnsupportedEncodingException
+    {
+        sysErr = System.err;
+
+        System.setErr(new PrintStream(errContent,false, "UTF-8"));
+

```

Comment:
remove blank lines at the end of methods

Reply:
Done